app/pages/admin/project-status/toggle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import AutoSave from '../../../components/auto-save';
+
+class Toggle extends Component {
+  constructor(props) {
+    super(props);
+    this.set = this.set.bind(this);
+    this.state = {
+      error: null,
+      setting: { }
+    };
+  }
+
+  set(property, value) {
```

Comment:
Directly modifying the state is an anti-pattern. Can this be reworked to use `setState`?